source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -63,34 +96,3 @@ The following function was tagged with `'codegen_method': 'private'`, because th
 - `trigger` parameter in `SendSoftwareEdgeTrigger`
 - `eventId` parameter in `WaitForEvent`
 - `powerlineFrequency` parameter in `ConfigurePowerLineFrequency`
-
-## attributes.py
```

Comment:
Note to owners: We have not modified any line. Just copy pasted these things under the first section of Changes wrt nimi-python metadata